samples/sampler/custom-addon/ColorPicker.js

```diff
@@ -24,11 +24,14 @@ const ColorPicker = ({colorPickerType}) => {
 	}, []); // eslint-disable-line react-hooks/exhaustive-deps
 
 	return (
-		<input
-			type="color"
-			value={globals[colorPickerType] || getDefaultColor(colorPickerType)}
-			onChange={handleChange}
-		/>
+		<div>
```

Comment:
Is this needed? Can we use fragment instead?

Reply:
we can use fragment